.readthedocs.yaml

```diff
@@ -9,6 +9,10 @@ sphinx:
 formats: [htmlzip]
 
 python:
-   version: "3.8"
    install:
    - requirements: docs/.sphinx/requirements.txt
+
+build:
+   os: ubuntu-20.04
```

Comment:
Is there any reason to prefer 20.04 to 22.04? I'd be tempted to specify the newer one, as it will probably be supported longer.

Reply:
changed to 22.04